js/templates/serverConfigForm.html

```diff
@@ -46,21 +46,19 @@
               </div>
             </div>
 
-            <div class="flexCol-8 borderRight0 custCol-border <% ob.errors.server_ip && print('invalid') %>">
+            <div class="flexCol-3 borderRight0 custCol-border <% ob.errors.server_ip && print('invalid') %>">
               <input name="server_ip" type="text" id="fld_server" placeholder="" class="fieldItem" required="" value="<%= ob.server_ip %>">
-              <div class="btn btn-txt btn-corner btn-cornerTR custCol-secondary ion-locked
-                          positionAbsolute tooltip <% if(!ob.SSL){ %> hide <% } %>  js-sslOn"
-                   data-tooltip="<%= polyglot.t('SSLIsOn') %>">
-                <%= polyglot.t('SSLOn') %>
-              </div>
-              <div class="btn btn-txt btn-corner btn-cornerTR custCol-secondary ion-unlocked
-                          positionAbsolute tooltip <% if(ob.SSL){ %> hide <% } %>  js-sslOff"
-                   data-tooltip="<%= polyglot.t('SSLIsOff') %>">
-                <%= polyglot.t('SSLOff') %>
-              </div>
+            </div>
+            <div class="flexCol-5 sslSwitch-radio ustCol-border <% ob.errors.server_ip && print('invalid') %>">
+              <% if(ob.SSL) { %>
+              <input type="radio" class="fieldItem" id="js-sslOn" name="sslSwitch" value="true" checked /><label for="js-sslOn" class="radioLabel"><%= polyglot.t('SSLOn') %></label>
```

Comment:
It would be better to have one set of radio buttons, and then inside each one, do 
` <% if(ob.SSL) { print(checked) } %>
For the true one, and !ob.SSL for the false one.